### PR TITLE
create the build infrastructure for nuget 4.0.1 rtm

### PR DESCRIPTION
**Pull request by rohit21agrawal**

### CODE REVIEW

File: .nuget/packages.config [hunk 1]
```diff
@@ -14,4 +15,11 @@
   <package id="VSLangProj150" version="1.0.0" targetFramework="net46" />
   <package id="xunit.runner.console" version="2.1.0" targetFramework="net45" />
   <package id="xunit.runner.msbuild" version="2.1.0" targetFramework="net45" />
-</packages>
+  <package id="xunit.runner.console" version="2.3.1" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net45" />
```
Comment (nkolev92) [inline .nuget/packages.config:19]: Why were these needed? 
newtonsoft.json, xdt, lucene etc. 

Them being part of the VSIX is the only connection I can make.

**Review by nkolev92 — APPROVED** [review]

File: .nuget/packages.config (hunk 1 above)
Comment (rohit21agrawal) [reply .nuget/packages.config:19]: yeah, the client projects in 4.0 need 6.0.4 version of newtonsoft.json and this is how we get it into the vsix.
same for lucene and xdt.

Comment (nkolev92) [reply .nuget/packages.config:19]: but this is based on the 4.0.0 version of the code that should've already had it there if that was the way

Comment (nkolev92) [reply .nuget/packages.config:19]: Dicussed offline :)

**Pull request merged**